public/usage-examples/graphics/draw_line_on_bitmap/draw_line_on_bitmap-1-map-route-oop.cs

```diff
@@ -0,0 +1,34 @@
+using SplashKitSDK;
+
+namespace DrawLineOnBitmap
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create a Window and bitmap for the map
+            Window window = new Window("Window", 400, 300);
+            Bitmap bitmap = new Bitmap("map", 400, 300);
+
+            // Fill background with white for map background
+            SplashKit.ClearBitmap(bitmap, Color.White);
+
+            // Draw the route line and start and end points
+            SplashKit.DrawLineOnBitmap(bitmap, Color.Green,
+                          100, 80,    // Starting point (x1, y1)
+                          300, 220);  // End point (x2, y2)
+            SplashKit.FillCircleOnBitmap(bitmap, Color.Red, 100, 80, 5);    // Start point
+            SplashKit.FillCircleOnBitmap(bitmap, Color.Red, 300, 220, 5);   // End point
+
+            while (!window.CloseRequested)
+            {
+                SplashKit.ProcessEvents();
+                // Draw the bitmap to the window
```

Comment:
This code comment is to different than what is in the other programs. Please update this.